### DISCUSSION TITLE
Hallo dewenni,

### DISCUSSION THREAD

**Discussion by Kenaschon** [dewenni/ESP32-Jarolift-Controller#7] [General]
geht das alles auch auf einem Wemos D1 mini?

Ich habe das "Ursprungsprogramm" auf einem Wemos laufen.

Grüße
Manfred

**Comment by dewenni** [comment]
Hallo Manfred,

nein das ist nicht möglich. Das Projekt ist bewusst auf einen ESP32 ausgelegt.
Ich empfinde den ESP8288 worauf auch der Wemos D1 Mini basiert, als nicht performant genug.
Der Webserver von dem original Projekt brauchte bei mir immer ewig bis die Seiten angezeigt wurden.
Es gibt aber auch von dem ESP32 eine sehr ähnliche Bauform und ich denke das zukünftig viel mehr ESP32 als ESP8266 eingesetzt werden.

Beispiel: [ESP32-Mini](https://www.amazon.de/diymore-Development-NodeMCU-Bluetooth-Entwicklungsboard/dp/B0D9LFM1MG/ref=sr_1_1_sspa?adgrpid=70013918526&dib=eyJ2IjoiMSJ9.osPumy2RE7TviGfW9UyYOsrvlRHX5hoYlVl3PZ707kRlqG6xik2kkN-0icJD040DciLPnbMAzgl_PF6dHwLrByP41uUDimmqRLJK2uJQVMAjtYF5njA-4QCUGpE5VHQKm6XTdwgpSppTkX7YXXkj6vtLJojSqzIkD1N2kAnAiFWGUp8BcMoij42NhTJbYNmFGjjBMkj2XyFAVQsWf39Aj22P9FXqEdAXaD1AsSOju1k.YkjXJdF3UsqZXGYjx9t0Z1DoPfGXoKdhowKr2nSghWs&dib_tag=se&hvadid=353094584366&hvdev=c&hvlocphy=9068376&hvnetw=g&hvqmt=e&hvrand=8649650927547531976&hvtargid=kwd-718540970767&hydadcr=8208_1722844&keywords=esp32%2Bd1%2Bmini&nsdOptOutParam=true&qid=1734783771&sr=8-1-spons&sp_csd=d2lkZ2V0TmFtZT1zcF9hdGY&th=1)

**Comment by Kenaschon** [comment]
Hallo dewenni,

habe es gerade auf einen ESP32 geladen.

Super Arbeit. Vielen, vielen Dank dafür.

Grüße
Manfred

**Reply by dewenni** [reply]
Freut mich!
Drück doch gerne noch den <Stern>. Das hilft das Projekt bekannter zu machen.
Grüße Sven
